pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -146,14 +146,21 @@ func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp
 	return common.Transform(ctx, manifest, instance, extra...)
 }
 
+// installedTransform mutates the namespace of all installed resources
+func (r *Reconciler) installedTransform(ctx context.Context, manifest *mf.Manifest, comp base.KComponent) error {
```

Comment:
Same with above, `installedTransform` just mutates the namespace so the function name should just be more specific.